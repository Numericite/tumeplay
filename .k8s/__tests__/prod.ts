//

import { getEnvManifests } from "@socialgouv/kosko-charts/testing";
import { project } from "@socialgouv/kosko-charts/testing/fake/gitlab-ci.env";

jest.setTimeout(1000 * 60);
test("kosko generate --prod", async () => {
  expect(
    await getEnvManifests("prod", "", {
      ...project("tumeplay").prod,
      RANCHER_PROJECT_ID: "c-5rj5b:p-s7trd",
    })
  ).toMatchSnapshot();
});
