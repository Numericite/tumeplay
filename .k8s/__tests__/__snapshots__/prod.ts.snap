// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`kosko generate --prod 1`] = `
"---
apiVersion: apps/v1
kind: Deployment
metadata:
  annotations:
    kapp.k14s.io/disable-default-ownership-label-rules: ''
    kapp.k14s.io/disable-default-label-scoping-rules: ''
    app.github.com/job: '5678'
    app.github.com/ref: refs/tags/v1.2.3
    app.github.com/repo: socialgouv/tumeplay
    app.github.com/run: '1234'
    app.github.com/sha: '0123456'
  labels:
    app: backend
    application: tumeplay
    owner: tumeplay
    team: tumeplay
  name: backend
  namespace: tumeplay
spec:
  replicas: 1
  selector:
    matchLabels:
      app: backend
  template:
    metadata:
      annotations:
        kapp.k14s.io/disable-default-ownership-label-rules: ''
        kapp.k14s.io/disable-default-label-scoping-rules: ''
        app.github.com/job: '5678'
        app.github.com/ref: refs/tags/v1.2.3
        app.github.com/repo: socialgouv/tumeplay
        app.github.com/run: '1234'
        app.github.com/sha: '0123456'
      labels:
        app: backend
        application: tumeplay
        owner: tumeplay
        team: tumeplay
    spec:
      containers:
        - image: ghcr.io/socialgouv/tumeplay/backend:v1.2.3
          livenessProbe:
            httpGet:
              path: /_health
              port: http
            initialDelaySeconds: 30
          name: backend
          ports:
            - containerPort: 1337
              name: http
          readinessProbe:
            httpGet:
              path: /_health
              port: http
            initialDelaySeconds: 30
          resources:
            requests:
              cpu: 300m
              memory: 256Mi
            limits:
              cpu: '1'
              memory: 1Gi
          startupProbe:
            httpGet:
              path: /_health
              port: http
            initialDelaySeconds: 30
          volumeMounts:
            - mountPath: /app/public/uploads
              name: uploads
          envFrom:
            - secretRef:
                name: azure-pg-user
            - secretRef:
                name: backend-sealed-secret
          env:
            - name: DATABASE_CLIENT
              value: postgres
            - name: DATABASE_NAME
              value: $(PGDATABASE)
            - name: DATABASE_HOST
              value: $(PGHOST)
            - name: DATABASE_PORT
              value: $(PGPORT)
            - name: DATABASE_USERNAME
              value: $(PGUSER)
            - name: DATABASE_PASSWORD
              value: $(PGPASSWORD)
            - name: DATABASE_SSL
              value: 'true'
      volumes:
        - persistentVolumeClaim:
            claimName: uploads
          name: uploads
      initContainers:
        - env:
            - name: WAIT_FOR_RETRIES
              value: '24'
          envFrom:
            - secretRef:
                name: azure-pg-user
          image: ghcr.io/socialgouv/docker/wait-for-postgres:6.26.2
          imagePullPolicy: Always
          name: wait-for-postgres
          resources:
            limits:
              cpu: 20m
              memory: 32Mi
            requests:
              cpu: 5m
              memory: 16Mi
---
apiVersion: bitnami.com/v1alpha1
kind: SealedSecret
metadata:
  annotations:
    kapp.k14s.io/disable-default-ownership-label-rules: ''
    kapp.k14s.io/disable-default-label-scoping-rules: ''
    app.github.com/job: '5678'
    app.github.com/ref: refs/tags/v1.2.3
    app.github.com/repo: socialgouv/tumeplay
    app.github.com/run: '1234'
    app.github.com/sha: '0123456'
  name: backend-sealed-secret
  namespace: tumeplay
  labels:
    application: tumeplay
    owner: tumeplay
    team: tumeplay
spec:
  encryptedData:
    MONDIALRELAY_ID: >-
      AgBIXbydPau0Vk+AbtrvcX3djGF1uFFQ12PWNOdqff5E2XvQ516WdU5L1Kh3UwikKyqp3PMLw/O7blp2V2Dfo3zVHT3W3ZYKmCtnpN0DzvLAUcN1XdqxkHOdro47IM0tD4DOMx3h4S9qrX0HJHkJb2hAi1ABCwUphXMZYwCCLYsBMVi6vlXGbHIJZe3qUYdMf/N0KypZKKm0Gjh1P6ltbXDypwkQrwbS57m+sRMVV3CL4PLAcrSM7EHEHvSKMa8aptEL6X4KZGM1U9AAbhWM8aQ+8aKGwe5uKGELd5y83xHxbZIjeEyuub4GfO/ZNMLAVSq8IjBt0f1yelMfzLtfHce5FONLr31AimAXsqCc7vzWrS4hDdsWsfu4zpucJiNO76RWsoBBuqo0/jprW++looufA8z18f/lqHfKTc2k58ArvKK+94p2TGw/RjanKj/jFou7PPKBybAFaiiAPdKyUHFRxK0tLgHv3EeZJgwwPeCEpVabskq2VF7XCLKqyutzdavkaJZMNkXnQvDptj4vIklhHx0+RfmJgBNpLyxNf7zpudD9wLyG47Zu/C2ZNIon3Oso3tOMN3Qt9pbDpjEiZKkcnDLu9Yi0Y+Y5mIze+jFrMD2kBTzBAZ3BDbNGj/Yi1U/yrgSKTV2FY3rlChF5HpO1WmcZ0GfIC/cYrkw12YtTOxLi500xlMJI0aFe0EdWNMh/C5hLx25PdA==
    MONDIALRELAY_SECRET: >-
      AgB7O4E/qUXYy+qCpsUEmUVwDaiBz6ca1wHcjKzKHHWeZXFlF1hMcY9npr7GI0OP+xt7grSmFvPMFrC1moAliNwIqQTtFWd7ZyMcn31VV6HDQcAEiLDeXWKE2kgjMLydaqF9RNN23ifDngmH6IvkTQTfM1BHtGd0OgGvlLPJoBxFoZCWbWF0b5Q8SxPedYozGMrebkJjgwb7gfB149ftV9b+IJoHGnt57xRdkNicL7Vgd0QRMkJ4AWm6UdTgUfqSs7qKV+QCICn6F5i8fVIhPoQMnDCyiA0Q2yBhe4NCBNfgzvAqMc/PPgAOsqvHOPMc6fuQ06EI3mUFUIWhwqOZR1+m76cTeObhhZxMESnW3bOnUIGEPfFnsRfkmxB0snDmQA0QnhLtgRz+SwY73vLekpyYgAAj627usU8cGxZTerdaho+70MFB1mJ0q9Vf8O4dg0CtTfaZVDo2Iz8TSfcHsxqcuC+eVB1Zk8RZ1rNXZqvyy2o0KZO637fUaoeQTzEctPit6sPIV5Bm5PkRlFrXtNJyfnMMLYcQMUeTulD29SB6yTMS6NkkdBsO6/BOgpd5BCBZSoNaT8HFDiPMzUEP+TcugHlgplvK+QDbM1jjui6Hai9QUjMZPyscGsW8rIiq23THje7dSzDRmRLCRCNeObJtv380Pr8+UWRL4xsHkyPfc4zpzt6g0v3mobkMK77FVWpRBp3QG7ZCIg==
    EMAIL_PROVIDER_PASSWORD: >-
      AgBnzI36LhT4NAjEw4h9YFFkoBXPD/v0qqngn6u2tMadi9JNkgsD9kfVm0PiSK1pvwOuqr4QGhARtR+Livo72OoVT5wRgEoGOIsRRCtkilfcraIaFezpbHQnI4ekJCOn5p03cul1ojiv8hVVh9S7jFxa7NQNxp6Bll/1I+PJ4MkPSKy/D7AVWt6475dDZkRobs7yQ8bYHSlcHqk9hljuE7VO36Dawnx8TWaH10TJufgtr+xhSEJu60iwwJJblI9uK2cHvOz57Qu4N3WIMDIYdwsA0jjcy99gPcqaGN5jUwZhwdkW8EUPpzmVV7CqegZHH65p+tZZ4HcxuopRDGEPlKAhYHcaLXLhCErOTPV8Nc4Ngos+sKQaPXPBf5qpyTkI4F0lxR54CP9MnbZVhDYt2lmVYUTGGJH0jAqKVpvtxaPFjk3IuC2e2AcXBlcol/vQNaUnpqwmmWSKHIZKgArX22vjQs8FNHNXzZnGSSCFxK2ImbGk5HTuAphfZ2VPeQeGhj3zXIslb5dCqnbXhhEYLxBP+xlOaQnCWqx9E45sEpp5rvd+OWdRb0usxm30HN+gr26lsfdoGX8k8nrWGBHiUSl1S+3qTmdkGbYj2A5cJu5Tyqm+MGy9v9GfTAcD467yHH+BNxF7Rj25ZpM9XYCrNrbnIAIZHvroyGZAIuPByjsuNt/v+z+U2Cm3zP5XN8l70VdpsSDaso+zmRB5Yg==
  template:
    metadata:
      annotations:
        kapp.k14s.io/disable-default-ownership-label-rules: ''
        kapp.k14s.io/disable-default-label-scoping-rules: ''
        app.github.com/job: '5678'
        app.github.com/ref: refs/tags/v1.2.3
        app.github.com/repo: socialgouv/tumeplay
        app.github.com/run: '1234'
        app.github.com/sha: '0123456'
      name: backend-sealed-secret
      labels:
        application: tumeplay
        owner: tumeplay
        team: tumeplay
    type: Opaque
---
apiVersion: v1
kind: Service
metadata:
  labels:
    app: backend
    application: tumeplay
    owner: tumeplay
    team: tumeplay
  name: backend
  annotations:
    kapp.k14s.io/disable-default-ownership-label-rules: ''
    kapp.k14s.io/disable-default-label-scoping-rules: ''
    app.github.com/job: '5678'
    app.github.com/ref: refs/tags/v1.2.3
    app.github.com/repo: socialgouv/tumeplay
    app.github.com/run: '1234'
    app.github.com/sha: '0123456'
  namespace: tumeplay
spec:
  ports:
    - name: http
      port: 80
      targetPort: 1337
  selector:
    app: backend
  type: ClusterIP
---
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  annotations:
    kubernetes.io/ingress.class: nginx
    certmanager.k8s.io/cluster-issuer: letsencrypt-prod
    kubernetes.io/tls-acme: 'true'
    kapp.k14s.io/disable-default-ownership-label-rules: ''
    kapp.k14s.io/disable-default-label-scoping-rules: ''
    app.github.com/job: '5678'
    app.github.com/ref: refs/tags/v1.2.3
    app.github.com/repo: socialgouv/tumeplay
    app.github.com/run: '1234'
    app.github.com/sha: '0123456'
  labels:
    app: backend
    application: tumeplay
    owner: tumeplay
    team: tumeplay
  name: backend
  namespace: tumeplay
spec:
  rules:
    - host: tumeplay-backend.fabrique.social.gouv.fr
      http:
        paths:
          - backend:
              service:
                name: backend
                port:
                  name: http
            path: /
            pathType: Prefix
  tls:
    - hosts:
        - tumeplay-backend.fabrique.social.gouv.fr
      secretName: backend-crt
---
apiVersion: bitnami.com/v1alpha1
kind: SealedSecret
metadata:
  annotations: {}
  name: azure-tumeplay-volume
  namespace: tumeplay
spec:
  encryptedData:
    azurestorageaccountkey: >-
      AgBCZ94S0NBkf4kpOzdv/okfFIislG2ZWF9L00kRySBfk8x69PkfIBSvfmVjQlCNCVdQTkd457GBIBONkT58vXPL/QHqhwF+lQJZiu0K9i4KCciVasiWg/WHAa98kZmAU3HXVaN0Lasat2Po+WYvh0IXBWMzEcBOWAZ//HgFPNmqtelfHH0WK8pliypT6BDbewmmP+9+5ZINO1i21H0MBd0tFPc/gJJ3QlbjcWcQOvmhnwgjA44huJnJhXXrphEDf5Qt9Vf0ZuqtpsFZy2AwSJiFAt/49BBHgXymdeWpOKF14lBAWsemIVFm+OEJhk3hR1r+fV/u+KHTD5E7bzXQyKYBWv4LGYkghvXYQsvfszBLCr+fzln8dci5PorymPOPY20mttia72rpTdPLpAlRaybIGExuJ5fIRuEqUjC5qEkmsuMjaH6NRxaG2zxVxrVjn1o1E18k6KCqVdlR8qSvzwwcgeLX1IDDKAjGxhALQY+G/9X5kdurVV7cgq5gk6yZOf/fKohBBFJEWzqEtEQhr5XvueQVN+8dRZgJVWSkny6V1d6Q1Yn7N+zOg1eRfsQFaWEzJj2WDd9iciOTfqtDLMwBM0PV3OzUcbmYQdJ24sr5lHsJivNc7ASL/3rSMNySUrh3RERsE6/BZG//XEYt6HD4AIH5XJjqL2fo5/XdcYXFjBBz21JnGMJDYNsy/OEArqoaJowPPuZgJpQisBR2xvcGOhY04wQJS7XBLOZr97PjygTHAtv8ZCQVZap/tOeFFVQ56tBu2utpS8aiipiyzlpdCZ3SMRsAwqJ+19/MdtItfeVMs7Jz0dg2
    azurestorageaccountname: >-
      AgBFQmJrPMSqXhokDwG7DUYxCAwxSQqdDkpJUBqoHynE4Bm7J22lmFjPLG5dnenavw/GUwhobCU2Gi6bjfohRDJZzLC4Gmhv5qWCVEfm1evUHm36Li47vbAFYmsua6v/50eqbuzzquowiyLCsh84XtUQHRu5LsYkFvRZb28XTACU/oRgelIatrmh3DwqWXV+LPbakQRPAoBEKU2uUbenyjiSTivokF5sBrHpNPgW12y2pgKfhp+gwpL/pI4YTt4ql0VK9OiLo16IhV04/pAjovNLAWoQeLOa4TM1rTa7SYlKkBNgtn1szoQJNu8hoeXqLmpoEeOyFzWhKjAm/e6ZNW9z0dD2j0oZb9k6QtVMStcLBDmPEbG66GghLlaaLWInk80onvJzmPYS0tqkmu2M/j/8XXSDGu5wFr+EjoWqoQZ34EwSytmWvk8UbebJYYaW1jJ3ASOj2lXur9tLk39f8kvoDjQa6rGsPfMZLS9u2EHWPx65D8p/vUPHwCrgrtyOFm99R7GnhHQNm/9nkodJEKVoc+0g0W/86/RpNFH+h31/Va0XASAz0USBXCO7njhLfngdxWRP7ag3voNNdQ/gvy2Ij+zvn8MkZPb0XT1XTKiJSZGb0gGWUrz8MQE3KWl4ow4E2mJRyWefGevMrEQjo1sCdBJzn3uEn8G5bBxIn18HWhit7PEAYptNqYu64S6YOUwu5T9N9iMtuuRE5cQ=
  template:
    metadata:
      annotations: {}
      name: azure-tumeplay-volume
      namespace: tumeplay
    type: Opaque
---
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  name: uploads
  annotations:
    app.github.com/job: '5678'
    app.github.com/ref: refs/tags/v1.2.3
    app.github.com/repo: socialgouv/tumeplay
    app.github.com/run: '1234'
    app.github.com/sha: '0123456'
  labels:
    application: tumeplay
    owner: tumeplay
    team: tumeplay
  namespace: tumeplay
spec:
  accessModes:
    - ReadWriteMany
  resources:
    requests:
      storage: 5Gi
  selector:
    matchLabels:
      usage: tumeplay-uploads
  storageClassName: ''
---
apiVersion: v1
kind: PersistentVolume
metadata:
  labels:
    usage: tumeplay-uploads
    application: tumeplay
    owner: tumeplay
    team: tumeplay
  name: tumeplay-uploads
  annotations:
    app.github.com/job: '5678'
    app.github.com/ref: refs/tags/v1.2.3
    app.github.com/repo: socialgouv/tumeplay
    app.github.com/run: '1234'
    app.github.com/sha: '0123456'
  namespace: tumeplay
spec:
  accessModes:
    - ReadWriteMany
  azureFile:
    secretName: azure-tumeplay-volume
    secretNamespace: tumeplay
    shareName: uploads
  capacity:
    storage: 5Gi
  persistentVolumeReclaimPolicy: Delete
  storageClassName: ''
---
apiVersion: bitnami.com/v1alpha1
kind: SealedSecret
metadata:
  annotations:
    app.github.com/job: '5678'
    app.github.com/ref: refs/tags/v1.2.3
    app.github.com/repo: socialgouv/tumeplay
    app.github.com/run: '1234'
    app.github.com/sha: '0123456'
  name: azure-pg-user
  namespace: tumeplay
  labels:
    application: tumeplay
    owner: tumeplay
    team: tumeplay
spec:
  encryptedData:
    PGDATABASE: >-
      AgCYlcvGKOw/hjj2k1/GwCsYFJtXvq5lZkNreMJ1tOwarQblVZPtKV1C+3oGuAqCm956rBK9/gsuf+XarG0moM7Hth/sOkzgB05TgXcyZRxEpHLYrDuOf0pFliTTQzYj8XC/wt2qzBHpYs/2b+NpPjVNGmswy9y2310Qmhcon3HIALAE6HOneIqTD3QmpZS7kVaId0Geiqr9H/bo8VyYgH9oGoGHIVioNvSBWBVzjgX24d63P8BlTq3fv1lVWwpegzGnQmrMb3+1j3coTZ1l8e/qZFKEp4QQEOEiDFtsY47wI5zxL/FqRxgK2ETNxgAXqib+PJBUKg+x60Dhnyix4JO6bjzdEkagGznv8nQmk3Bs/wHCXHL+PkpiM40yehDt8D1wlHkhsV2U1ooToKnVvcwYcblwPG6WzBmNL5anOpGNDZYTdE4kE6g2ta5ml8Vhk6C5CSLE98ks3cQ+aCIU+6znW9xOAFQPYIw6aCSyKK28LI/mDxaaiX0EMYHNXSmri1bupX9x9XmAyN4N51trzYhdbLJscGRmbx0NFqcu2id5HgX1LhgeWqWojwxHbD7fJaYYuAsHTsJuyXZSxfSFOjNzkkQPN54p9pDlGPbLBxhjuJra/pJnp97BQrvD14WU6U5OgYgVahD66d7IhpW7ToTrcm3zHXmrwxCOFoAXwnmhM80BjqxLq6Orm5LqVk77yvVs3Ojk
    PGHOST: >-
      AgAJsDlRVU7U7YfosYy9CAqxHlWMTYeKboZ0X9nT9w98x4Ml5nR8Oh+jn/66Hgw/amVvKShmMlH1gwPOItaskxU5K2mNdRKutxJPwSh1j1z69A+B+09AGNLP2kVr3eIUosdqyh7Wyb/rbvxe83nb3ddKmWytv4fc7q/GwypYrOjtGGpFpZxkC8lH+QEpVOH0jv6mgYDjopw7oYMp25vBPepOHRs4nriMUqTEO+9Tp2i0d3Hl1FHmxwndAeVto2NSdSv5ZUvGFT/yHr5nfO6vnKx+M6a5yDnlWvoBFNuneCYJHKIxGQ2hkiM8hpBrzNZA/VI4hdYvmH2I/PskPimMf4K1cRjUjxJzsYN5JLVV+UC02ch0kM8wzKyVVYKel9lpFO0zppCH2+Qd8/XTi6KmrRO3cx6eQdPFnG1egzKoELmQ8EaELfP6b8PKAWrN68Jul0xmE/vT1fI+1XDEBcxn1fx7BRCdt0et6dVETr50/7J5/IuRFTLZ3+iUJuuCTTm63i2Px87OUZAk1uL6+nAZvzhStvVo0Uc1IB7lxl/ScSo4SxhszJ7U1aOxlj2lk2OReDAeGDVhMHninSTcOXqqxChkXuvgjItLJhqp5KMmlRZsfp6Kx0IUvxNQr161I5AoNJxfCWY/VuzvAEiy6IKfSqzJSZIrBiD9XoZFrLp/of0OYtVo6RYr0PqbI29MsrxGGQvJPVgzHhUjmeqXhiSyG0McUjduH1rxgMBt+2xQMZ011eWUQW1YfT00DAsqcXU=
    PGPASSWORD: >-
      AgBtonDwX7zu0d4FDNuBJwl8HO0K6ev+8dQhxUnUtMYEjNYNA5fGkiC180WSZHDDrcY6n1dtZM8t6YqQodsNG24JnHzvA6i0ojyY0xwmKpAvL5zqwU6bSBd6c4NF5/VmBH5wy7KAh+4evCfXFyRxsuPQNB6t1hh9YDtO96qMfU1lNRlCyHepcw9t8I498jUIfmbKP7iuB9aSGQVm04oKpFr1kQcBGfZS1tVuURN9gr9ZGtjYzN0CAcyLMfToRfJovR0+1ot4DNZURM3rZy23QUj737gQTYDm5QSII/IJeuxXhgA21JIkqcE/VrajYuEAVVAlVk/ABRu6DXmNEaupslTBjsZaKf79AjMqpX4MvDBat7efDrrCMT9acq1/H+BgbRkHt743aF7uXij3ntEGaJ1RvqnVK7/bBx1OEuCGUrVsRBKLMLJsWt/oycNr/DRt+KA1tfLApsXplmrCYnj1NckJB4E1JQg7TLMizSqTN8fsYC+oPrgiJye2XK1/UKjDERGCw7a0qPIGXSZhinw4fK59mFkXNbWJoONy1pMfAOCyqt9NQah3Ms0azF9SwJL3a0CzWSXRexu3Jn0WtzOEQ5y0aUHo6wBKHF34jpUTzBXQkANzPvdIwpgQX9/tiCH0ZSLjUomDOOyCxr/YqojMAohz8Yj3weY3OYu4XLuJpyqjGlvQRKF7BXkGVk22NJYIFbqoYjeADN5xgV1UDBF2ehCVCcbCF0KvISlBwtef3lXrZ+D+mJV1aHM=
    PGUSER: >-
      AgBOxT3JuUVzkaxarx7RwvkNbMV+7INBQ6wNetDZCDPkIxc8Bem8JkUxvNrOMX2uL41V/8l/OGohxl94XUw9WO/YppWU8GIcva5RE16tDbeK9v2BOLi38hFR856FXIZW8qtvdfkufIkyjihX5Hvdbss0i1/73SYeu6J8HVtLQz/AFLgqN1T7bQ4g2qf5B2QiKelsxCnUb2nHRvNufy5DbZu8UXXDqW9r4mHuNE2IkFXd6t3bKRbcqhjzhvqwfXA4P2T0vsugwqgJUHrwCO3Axe/jjVROkd+GamF79NBx81Vb5+N0CCG6zIYyXp1F4A7+RlKkyCAXK1iiRoIULZ/12ono/RmtiKQnXf6U+m6hiNHYQMnurqSryH82HNKWDevU377bDAaZHONXF0ND/wLOZbBZ9oR+E7Sh9yBlmVtW3Mp5HSy3iNRllZA3sLeSpBTb7nQiYeLl3kte9OvjR4G2VXdHOgctZd71v648Ngwy8otoCYm6JbxRvoaqMDj6KMJaO+UnTzWEzEokh+Ot9cgTL4+eEAI+gVCAH8rkgxT1TirRcCMAVsgY1vviySQ4aR0A7NLAbgiE3Nf9TD9blKBBXB7xTBUq6sQg3Cg/JLG//v8+AA5+WSVtOuZRMgNd31YPxUuB3/eG1505FAUZVgf1YRxYwqDcmHD31D2s/z5N99azQDrD8DjrAHzHwy6/qyQlInC5K31z0oBUqy3CtKoAmDZWNayaBzcV
    PGSSLMODE: >-
      AgCN9WYs2P2MfUlXtZktnJMjwseUAkbRwF+keBIyRimVmByVFve9y72QcoXBUpoEW1uy1T7M2ZK17SkGbYN35XjVSmhctW2QloLaC6dkxPJMotDXB+FFBsGxNNWkGhWjL7XFoKr/Jtt90e/rnwNZCJjEZlQvrFBt9Atm43WdIXZT7yEXIgcS6grnZaoJqXFkikMDypyl8yDKj4e103/WusTLUPaudpaWK67t0fXGXYlu0XsTzIxZLIUCdFe4VCEsyX2KY1ewTL/VNz+QpSQmEnS/xE+OZvsyL4/rIX+B7ekJx2S6yNxmk63dzKJAIuQ5ppUKqtIsyk7vDtjBlzVBbt8Gfky/SAJ0fsTQ4K/kjVh/3M3wQjr1fid7xcIPPmGrcWxbo41dIYQ6XQ2ZypQL015u48FSFRon1Xo9Y89IW/hMjXZ73gPZX3XEYkDNvkbpJ0PoSguEazFr01cns7oVLZqlw/ntwfCUNsuqeLTR0bUXAl+94gRFavuzCWkCZ3I0VoMDJyQozA1d3DXiP7xymRonvFu/qt9YnZj9+ZheA+WEm2LEGgbRtkc/E0z4ArnGEeOWxxvzkOE/NiASJBBm00CUCwFj0MccblE4bq6spbmwrqtqQXdbt1dSIVr3TEVW7dfXOiTTyVJSpJiJeH4IvSbEADW3bUuljww8vj7yHyiqNhaXZYCfYGj1JH0HobeHU75wHbl6eD2l
  template:
    metadata:
      annotations:
        app.github.com/job: '5678'
        app.github.com/ref: refs/tags/v1.2.3
        app.github.com/repo: socialgouv/tumeplay
        app.github.com/run: '1234'
        app.github.com/sha: '0123456'
      name: azure-pg-user
      labels:
        application: tumeplay
        owner: tumeplay
        team: tumeplay
    type: Opaque
"
`;
